azurestack/import_arm_loadbalancer_backend_address_pool_test.go

```diff
@@ -0,0 +1,34 @@
+package azurestack
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+)
+
+func TestAccAzureStackLoadBalancerBackEndAddressPool_importBasic(t *testing.T) {
```

Comment:
Not a blocker, but there is a push to add the import tests as a final step on the regular ones, feel free to do the same as you move resources over as it'll shorten test times & load on the env.